### ISSUE TITLE
Object names are inconsistent

### ISSUE THREAD

**Issue by denes44** [Dolibarr/dolibarr#32098]
### Bug

One module using multiple names in different places, which makes it hard to do things programmatically very hard.
Usually it just two names, one for the API, and one for the URL, or sometimes IDs, but I found a case, when there is a third name, which is nonsense now...

### Dolibarr Version

20.0.2

### Environment PHP

8.2.24

### Environment Database

MySQL or MariaDB 10.9.8-MariaDB-1:10.9.8+maria~deb11

### Steps to reproduce the behavior and expected behavior

The `Shipments` module are called `shipments` in the API, `expedition` in the URL if I want to make a link for example.
But if I query the API in the `linkedObjectsIds` it's called `shipping`. Why?

To make it even more inconsistent, in the `linkedObjectsIds` the `Orders` module are called `commande`.
This makes it very hard to use these information in third-party software, and require a lot of conversation tables...

**Comment by JonBendtsen** [comment]
Yeah, that confused me as well in the beginning. I think it is a leftover from being a French project that is in the process of internationalization.

**Comment by denes44** [comment]
it is still present on latest stable version
